src/compile.rs

```diff
@@ -11,7 +11,11 @@ pub fn script_gen(location_command: &str, hook: &str, hook_config_file: &str) ->
 #!/bin/sh
 # {}
 # Hook created by precommit
-{} run {} {}
+if  git rev-parse -q --verify MERGE_HEAD; then
```

Comment:
Can you do that from inside the tool itself, so you have more control (and don't change hook script, which is nice from backwards compatibility point of view)?

Reply:
Yea I could add arg parameter for this

Reply:
No, I mean, can you get that info in the tool itself, without `get rev-pase`?

Reply:
I probably can.